src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/ref/Microsoft.Extensions.DependencyInjection.Abstractions.cs

```diff
@@ -39,7 +39,7 @@ public partial interface ISupportRequiredService
     {
         object GetRequiredService(System.Type serviceType);
     }
-    public delegate object ObjectFactory(System.IServiceProvider serviceProvider, object[] arguments);
+    public delegate object ObjectFactory(System.IServiceProvider serviceProvider, object[]? arguments);
```

Comment:
Can the arguments themselves be null?  If so, this needs to be `object?[]?`.

Reply:
(Or should it be `object?[]`, meaning arguments itself can't be null, but it contain nulls?)

Reply:
I think this needs to be `object?[]?` - that is (1) no arguments at all, (2) any argument can be null.